Client/Frontend/Strings.swift

```diff
@@ -188,6 +188,20 @@ extension Strings {
     public static let UnableToAddPassErrorDismiss = NSLocalizedString("AddPass.Error.Dismiss", value: "OK", comment: "Button to dismiss the 'Add Pass Failed' alert.  See https://support.apple.com/HT204003 for context on Passbook.")
 }
 
+// Night mode
+extension Strings {
+    public static let MenuNightModeTitleString = NSLocalizedString("Menu.NightModeAction.Title", value: "Night Mode",  comment: "Label for the button, displayed in the menu, used to toggle night mode on or off")
+    public static let MenuNightModeTurnOnTitleString = NSLocalizedString("Menu.NightModeTurnOnAction.Title", value: "Night Mode",  comment: "Label for the button, displayed in the menu, used to turn night mode on")
```

Comment:
Is this supposed to be 'Night Mode On' or just 'Night Mode'? Maybe copy-pasta error?